content/src/main/content/jcr_root/apps/acs-commons/rte-plugins/source/dialog-plugin.js

```diff
@@ -0,0 +1,367 @@
+/*
+ * #%L
+ * ACS AEM Commons Package
+ * %%
+ * Copyright (C) 2013 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ *
+ * A sample component dialog using the Touch UI Multi Field
+ * Note the usage of empty valued acs-commons-nested property
+ *
+ */
+(function ($, $document, Handlebars) {
+    "use strict";
+
+    var _ = window._,
+        Class = window.Class,
+        CUI = window.CUI,
+        REQUESTER = "requester",
+        GROUP = "acs-commons",
+        INSERT_DIALOG_CONTENT_FEATURE = "insertDialogContent",
+        INSERT_DIALOG_CONTENT_DIALOG = "insertDialogContentDialog",
+        ACS_CUI_TOOLBAR_BUILDER,
```

Comment:
perhaps more a style issue than anything critical, but I'd rather we didn't use a constant styling for non-constant values. i.e. this just should be AcsCuiToolbarBuilder